cypress/integration/dib-travel/company-employees/groups.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { Groups } from '../../../models/company-employees/groups';
+
+describe('Company Employees - Groups Page', () => {
+  let groupsDetails: Groups;
+
+  before(() => {
+    cy.fixture('company-employees/groups').then((group) => {
+      groupsDetails = group;
+    });
+  });
+
+  // Regular User
+  it('should allow regular user to add new group', () => {
+    cy.login();
+    cy.get('dib-layout dib-hamburger-icon').click();
+    cy.get('[routerLink="/people-management/groups"]').click();
+    cy.get('dib-people-management dib-groups ui-button button').contains('Add Group').click();
+    cy.get('.cdk-overlay-container dib-group-dialog input[placeholder="group name*"]').type(groupsDetails.name);
+    cy.get('.cdk-overlay-container dib-group-dialog dib-assign-members .members mat-checkbox').click();
+    cy.get('.cdk-overlay-container dib-group-dialog ui-button button').contains('save').click();
+    cy.get('dib-people-management dib-groups .body').should('contain', groupsDetails.name);
+  });
+
+  it('should allow regular user to edit created group', () => {
+    cy.login();
```

Comment:
You repeated this too much:

```
cy.login();
cy.get('dib-layout dib-hamburger-icon').click();
cy.get('[routerLink="/people-management/groups"]').click();
```

Please do same as I commented in related PR.

Reply:
I left this because I have user and agent login and if I only add user login inside beforeEach I will only be able to login as a regular user. Do I need to delete whole agent test or there is a better solution for this?

Reply:
It's better to fully separate tests with agent login in a separate file.

- `cypress/integration/dib-travel/company-employees/groups.spec.ts` - regular tests with user login
- `cypress/integration/dib-travel/company-employees/groups-agent.spec.ts` - regular tests with agent login

If you have the same or similar logic for the user and for an agent, we can create shared commands or something similar. But please leave that for letter improvements, for now just copy/paste code block and make selector adjustments